ExampleReconstruction/OSEM.par

```diff
@@ -0,0 +1,39 @@
+OSMAPOSLParameters :=
+; example file for OSEM in the STIR-GATE-Connection
+
+  objective function type:= PoissonLogLikelihoodWithLinearModelForMeanAndProjData
+  PoissonLogLikelihoodWithLinearModelForMeanAndProjData Parameters:=
+
+    input file := ${Prompts}
+
+    projector pair type := Matrix
+    Projector Pair Using Matrix Parameters :=
+      Matrix type := Ray Tracing
+      Ray tracing matrix parameters :=
+        number of rays in tangential direction to trace for each bin := 10
+      End Ray tracing matrix parameters :=
+    End Projector Pair Using Matrix Parameters :=
+
+    ; additive projection data to handle randoms etc
+    additive sinogram := ${AdditiveSinogram} 
+
+    ; norm and acf
+    Bin Normalisation type := From ProjData
+    Bin Normalisation From ProjData :=
+      normalisation projdata filename:= ${MultFactors}
+    End Bin Normalisation From ProjData:=
+
+    subset sensitivity filenames:= subset_sens_%d
+    recompute sensitivity := 1
+
+    xy output image size (in pixels) := -1
+
+  end PoissonLogLikelihoodWithLinearModelForMeanAndProjData Parameters:=
+
+  ; Number of subsets should usually be a divisor of num_views/8
+  output filename prefix := ${SGCPATH}/ExampleReconstruction/OSEM
+  number of subsets:= 8
+  number of subiterations:= 40
```

Comment:
provide these as parameters I guess (wouldn't work for the mMR for instance).  Can still hardwire them in the script (but most people will be more comfortable in changing that)

Reply:
I moved it out to the script with the option to set them